src/mito2/src/error.rs

```diff
@@ -877,6 +878,14 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+
+    #[snafu(display("Fetch TTL timeout"))]
+    FetchTtLTimeout {
```

Comment:
I think we can define a timeout error so we can reuse it in other cases.
```suggestion
    #[snafu(display("Timeout"))]
    Timeout {
```